requirements_minimal.txt

```
@@ -6,5 +6,5 @@ lameenc>=1.2
 openunmix
 pyyaml
 torch>=1.8.1
-torchaudio>=0.8,<2.2
+torchaudio
```

Any reason for removing the pinned version? I assume that the initial lock was here to prevent against a regression or API changes, which might now have been fixed since. If everything is working well, I would suggest you at least pin forward, for example

```suggestion
torchaudio>=2.7.1
```

---

Created #1 for tracking

---

> Created #1 for tracking

This should be #2